package.json

```diff
@@ -48,20 +48,20 @@
   "homepage": "https://github.com/mojaloop/email-notifier#readme",
   "devDependencies": {
     "chai": "4.2.0",
```

Comment:
Dont believe your tests require mocha and chai

Reply:
Please move tape and tapes from dependencies to dev dependencies

Reply:
Performed per review.